src/vsr.zig

```diff
@@ -142,6 +142,8 @@ pub const Operation = enum(u8) {
     root = 1,
     /// The value 2 is reserved to register a client session with the cluster.
     register = 2,
+    /// The value 3 is reserved for reconfiguration request.
+    reconfigure = 3,
```

Comment:
Didn't find a better name for the request here, the problem with reconfiguration is that configuration is too generic. On the plus side, this maintains the invariant that all reserved operations start with r!

Reply:
From today's chat with Joran --- an important point here is that this is our public visible interface, rather than just our private API. In the docs, we do want to call this "reconfiguration" as that's the appropriate term here.

Reply:
Good point; I hadn't considered that.